lib/dotiw/time_hash.rb

```diff
@@ -11,8 +11,17 @@ def initialize(distance, from_time = nil, to_time = nil, options = {})
       self.options    = options
       self.distance   = distance
       self.from_time  = from_time || Time.now
-      self.to_time    = to_time   || (self.from_time + self.distance.seconds)
+      self.to_time    = to_time   || (@to_time_not_given = true && self.from_time + self.distance.seconds)
```

Comment:
This was meant for a quick (one-line) way to flag that ```to_time``` was not supplied as a parameter. It is an intentional assignment, but can be refactored if misleading.